lib/gear_application/gear_application.ex

```diff
@@ -28,8 +27,10 @@ defmodule Antikythera.GearApplication do
   alias AntikytheraCore.Config.Gear, as: GearConfig
   alias AntikytheraCore.GearLog.Writer
 
+  @type child_spec :: [:supervisor.child_spec | {module, term} | module]
```

Comment:
This should represent spec of 1 child.
```ex
@type child_spec :: :supervisor.child_spec | {module, term} | module
```
Then use this for parameter types as `children :: [child_spec]`.
We may also define `@type child_specs :: [child_spec]` but we probably don't need it.

Reply:
fixed in c9f159e